CommonCompilerConfig.cmake

```diff
@@ -36,35 +36,37 @@
 #                               source files; e.g., for cmake `add_definitions()`
 # COMMON_COMPILER_DEFINITIONS_STR: string variant of COMMON_COMPILER_DEFINITIONS
 #
-# COMMON_C_FLAGS_UNOPTIMIZED: string of common C flags without explicit optimization flags.
+# EXTERNAL_C_FLAGS_UNOPTIMIZED: string of C flags without explicit optimization flags for 3rd-party sources
```

Comment:
This isn't too important, just mention it here because it is easy to do for comments.